armory/data/pytorch_loader.py

```diff
@@ -0,0 +1,18 @@
+import torch
+
+
+class TFToTorchGenerator(torch.utils.data.IterableDataset):
+    def __init__(self, tf_dataset):
+        super().__init__()
+        self.tf_dataset = tf_dataset
+
+    def __iter__(self):
+        for ex in self.tf_dataset.take(-1):
+            x, y = ex
+            if isinstance(x, tuple):
```

Comment:
is this logic to handle adversarial datasets? Maybe a comment would make this a bit more clear

Reply:
referring to `if isinstance(x, tuple):` in my previous comment